disnake/flags.py

```diff
@@ -328,6 +373,11 @@ class SystemChannelFlags(BaseFlags):
             Returns a new SystemChannelFlags instance with all enabled flags from both x and y.
             (Using ``|=`` will update in place).
 
+            .. versionadded:: 2.6
+        .. describe:: SystemChannelFlags.y | SystemChannelFlags.z, SystemChannelFlags(y=True) | SystemChannelFlags.z
```

Comment:
Maybe this should go next to the `~SystemChannelFlags.y` docs at the end of the list? I'm aware this is right below the `__or__` docs, but that might be more confusing than the other way.

Reply:
https://github.com/DisnakeDev/disnake/pull/615/commits/1db2ed6c123f4fec0970aa921954ce3621184486